src/assets/js/fluent.js

```diff
@@ -57,6 +61,9 @@ class Fluent {
             });
             return true;
           }
+          if (inlineTags.includes(token.tag)) {
```

Comment:
<Opinion> I prefer to have all conditional returns of the same value under the same conditional. I feel like this conditional and the one above are super related and can be refactored to include one ```return true;``` statement

Reply:
That's true, I hacked this in without thinking about it too much, but now I'm looking at it again it's (kinda) possible that as written this would allow translators to add unintended attributes to these whitelisted `inlineTags` (which we don't want). I'll rewrite this :)

Reply:
Actually I discovered this couldn't happen, because of the sanitizing that the library was already doing, but I've added some tests to verify this.